packages/x-data-grid/src/hooks/features/columnHeaders/useGridColumnHeaders.tsx

```diff
@@ -69,24 +70,10 @@ const SpaceFiller = styled('div')({
   },
 });
 
-export const GridColumnHeaderRow = styled('div', {
-  name: 'MuiDataGrid',
-  slot: 'ColumnHeaderRow',
-  overridesResolver: (props, styles) => styles.columnHeaderRow,
-})<{ ownerState: { params?: GetHeadersParams; leftOverflow?: number } }>(
```

Comment:
[This test breaks](https://github.com/mui/mui-x/blob/next/packages/x-data-grid/src/tests/layout.DataGrid.test.tsx#L1024-L1034) if this slot doesn't have an `ownerState` prop, and I don't understand why. Removing the slot stuff made the test pass. Passing an empty `ownerState={{}}` also made it pass, but didn't felt clean.

Reply:
Did you manage to fix it? All tests pass

Reply:
Well I removed the slot stuff, so yes but the slot `ColumnHeaderRow` doesn't exist now. Is that ok?

Also why is that test failing? Isn't that override targetting the `root` slot of the datagrid?

Reply:
I have set an empty `ownerState` for now. The slot is back for backwars compat, and we can adjust the `ownerState` thing later.

Reply:
I have restored the `ownerState` thing.